corehq/apps/app_manager/xform.py

```diff
@@ -976,20 +979,37 @@ def _get_label_ref(self, prompt):
         return None
 
     def resolve_path(self, path, path_context=""):
```

Comment:
If `path` is now a ConditionalCaseUpdate I would rename the param to `update` or something.

Reply:
I clicked add too soon. Stylistic comment: If both ConditionalCaseUpdate objects and strings are supported, you could make that more apparent and shorten the code a bit by doing something like `path = getattr(path, "question_path", path)` instead of the try/except and the param rename.

Reply:
👍